src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -555,40 +555,22 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('cache')
                     ->info('Cache configuration')
-                    ->fixXmlConfig('adapter')
+                    ->fixXmlConfig('pool')
                     ->children()
-                        ->arrayNode('adapters')
+                        ->arrayNode('pool')
                             ->useAttributeAsKey('name')
                             ->prototype('array')
                                 ->beforeNormalization()
-                                    ->always(function ($v) {
-                                        if (!isset($v['options'])) {
-                                            $v['options'] = array();
-                                        }
-
-                                        foreach ($v as $key => $value) {
-                                            if (!in_array($key, array('type', 'name', 'options'))) {
```

Comment:
I don't get why we need to move options into the "options" subkey. Let'r remove this if it's not required.